### PR TITLE
Apply shadow-bevel to Toast component

### PR DESCRIPTION
**Pull request by sam-b-rose**
Part of https://github.com/Shopify/polaris-summer-editions/issues/711

Note: We are still using `--p-shadow-xl` here. Does this still apply even with the updated bevel styles?

## Default

| Before | After |
| --- | --- |
| before-default | default |

## With Error

| Before | After |
| --- | --- |
| before-with-error | with-error |

### CODE REVIEW

**Review by aaronccasanova — APPROVED** [review]
Code updates look good! Let's wait for @sarahill's feedback on the usage of the existing `shadow-xl` token before merging 🙏

File: polaris-react/src/components/Frame/components/Toast/Toast.scss
Comment (sarahill) [inline polaris-react/src/components/Frame/components/Toast/Toast.scss:13]:
```suggestion
  box-shadow: var(--p-shadow-lg);
```
let's go with `lg` for the shadows

**Review by sarahill — APPROVED** [review]
Looks good! made one suggestion to update the shadow to `lg`. 🎉

**Review by chloerice — APPROVED** [review]

File: polaris-react/src/components/Frame/components/Toast/Toast.scss
Comment (sam-b-rose) [reply polaris-react/src/components/Frame/components/Toast/Toast.scss:13]: Done! Looks good 💯 

Screenshot 2023-07-07 at 1 51 50 PM

**Pull request merged**
